transcripts/share-apis/definition-diffs/standard-ability-diff.json

```diff
@@ -52,7 +52,11 @@
             },
             {
               "annotation": null,
-              "segment": " "
+              "segment": "\n"
```

Comment:
do we know why this changed?

Reply:
Yup; I added a manual break in ability definitions;

Previously:

```
ability Blah where foo : Nat -> Nat
```

Now:

```
ability Blah where
  foo : Nat -> Nat
```